### PR TITLE
Update otelcol.exporter.otlp.md

### LINKED ISSUE #4186: Docs feedback: /docs/sources/flow/reference/components/otelcol.exporter.otlp.md
There is an issue on the documentation for the `queue` and `retry` blocks where the actual river mapping is the following:
- [`queue`](https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.exporter.otlp/#queue-block) => [`sending_queue`](https://github.com/grafana/agent/blob/v0.34.1/component/otelcol/exporter/otlp/otlp.go#L34)
- [`retry`](https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.exporter.otlp/#retry-block) => [`retry_on_failure`](https://github.com/grafana/agent/blob/v0.34.1/component/otelcol/exporter/otlp/otlp.go#L35)

Do you prefer a PR on the documentation to use the coded value or on the code to use the documented value ?

### PR DESCRIPTION
**Pull request by rgnvldr**
Docs don't align with code https://github.com/grafana/agent/blob/7878e6e01c3db150e875861763c095a5707552b1/component/otelcol/exporter/otlp/otlp.go#L33)

Fixes #4186

### CODE REVIEW

File: docs/sources/flow/reference/components/otelcol.exporter.otlp.md
Comment (clayton-cornell) [inline docs/sources/flow/reference/components/otelcol.exporter.otlp.md:128]:
```suggestion
### sending_queue block
```

File: docs/sources/flow/reference/components/otelcol.exporter.otlp.md
Comment (clayton-cornell) [inline docs/sources/flow/reference/components/otelcol.exporter.otlp.md:135]:
```suggestion
### retry_on_failure block
```

**Review by clayton-cornell — COMMENTED** [review]
Updated the headings as well.

@grafana/grafana-agent-maintainers can someone from the team double check this change before I merge.

**Review by rfratto — APPROVED** [review]
LGTM from my side after:

* Clayton's feedback is addressed, and 
* The same change is reflected in the two other pages that have the same problem

**Pull request merged**
